solidity/utils/test_deployment.js

```diff
@@ -308,9 +308,18 @@ const run = async () => {
                 }
             }
 
-            const deployedConverterType = {1: 'LiquidityPoolV1Converter', 3: 'StandardPoolConverter'}[type];
-            const deployedConverter = deployed(web3, deployedConverterType, converterBase._address);
-            await execute(deployedConverter.methods['addLiquidity(address[],uint256[],uint256)'](tokens, amounts, 1), value);
+            const contractName = {
+                1: 'LiquidityPoolV1Converter',
+                3: 'StandardPoolConverter'
+            };
+
+            const funcSignature = {
+                1: 'addLiquidity(address[],uint256[],uint256)',
```

Comment:
My previous comment couldn't be applied here, but that's understandable. Hopefully, we'd remove this in the future as well.

Reply:
Yup, and we can use a similar trick as the one that I've mentioned above, but it wouldn't yield any significant improvement to the current implementation in this script.

Reply:
Yeah, but if we can limit this only here - it'd be better